addon/components/tooltip-on-element.js

```diff
@@ -430,6 +440,16 @@ export default EmberTetherComponent.extend({
     this.sendAction('onTooltipShow', this);
   },
 
+  stopTether() {
+    run.schedule('afterRender', this, function() {
+      this.get('_tether').disable();
+    });
+  },
+
+  startTether() {
+    this.get('_tether').enable();
+  },
```

Comment:
could you add tests to check that the tether is enabled when we expect it to be?

```
isTetherEnabled: computed.alias('_tether.enabled'),
```